schema.go

```diff
@@ -45,6 +46,7 @@ var (
 	ipAddrType     = reflect.TypeOf(netip.Addr{})
 	urlType        = reflect.TypeOf(url.URL{})
 	rawMessageType = reflect.TypeOf(json.RawMessage{})
+	uuidType       = reflect.TypeOf(uuid.UUID{})
```

Comment:
IMO users should be able to inform which uuid library they are using, as a google library might not be their first option. @NicoleStrel What do you think?